timesketch/lib/tasks.py

```diff
@@ -26,6 +27,10 @@
     pass
 
 from timesketch import create_celery_app
+from timesketch.lib.datastores.elastic import ElasticSearchDataStore
```

Comment:
nit: In all the comments, Elasticsearch is used. But this lib is ElasticsearchDataStore.